Homework_3/Task_3.py

```diff
@@ -0,0 +1,11 @@
+#Задайте список из вещественных чисел. Напишите программу, которая найдёт разницу между максимальным и минимальным значением дробной части элементов.
+num_list = [1.1, 1.2, 3.1, 5, 10.01]
+max_num, min_num =round(num_list[0] % 1 , 2), round(num_list[1] % 1 ,2)
```

Comment:
```suggestion
max_num, min_num =round(num_list[0] % 1 , 2), round(num_list[0] % 1 ,2)
```
Если не принять за минимальный элемент 1 элемент и 8 строке, сделать elif вы рискуете пропустить проверку этого элемента.